docs/api/rest_api/rest_api_reference/input/ez-product-products.raml

```diff
@@ -156,18 +156,14 @@
                     Accept:
                         description: If set, gets price in given currency in JSON format.
                         example:  |
-                            application/json
-                    Content-Type:
-                        description: If set, the updated price is returned in JSON format.
-                        example:  |
-                            application/json
+                            application/vnd.ibexa.api.Price+json
                 responses:
                     200:
                         description: OK - gets price in given currency.
                         body:
                             application/json:
                                 type: Currency
```

Comment:
It returns a `PriceInterface`
- https://github.com/ibexa/product-catalog/blob/v4.6.0/src/bundle/Resources/config/routing_rest.yaml#L373-L376
- https://github.com/ibexa/product-catalog/blob/v4.6.0/src/bundle/Controller/REST/PriceController.php#L47-L51

Reply:
A `Price` and only that type according to [PriceGetWebTest](https://github.com/ibexa/product-catalog/blob/v4.6.0/tests/integration/REST/PriceGetWebTest.php#L18-L22)